unittests/test_e.py

```diff
@@ -721,10 +725,11 @@ def test_ekops():
     cs.ekcls(handle)
 
 
-def test_ekopw():
-    ekpath = os.path.join(TEST_FILE_DIR, "example_ekopw.ek")
+@checking_pathlike_filename_variants("path_type_variant")
+def test_ekopw(path_type_variant):
+    ekpath = path_type_variant(os.path.join(TEST_FILE_DIR, "example_ekopw.ek"))
     cleanup_kernel(ekpath)
-    handle = cs.ekopn(ekpath, ekpath, 80)
+    handle = cs.ekopn(ekpath, str(ekpath), 80)
```

Comment:
All of the other tests create a string and then make a path variant out of it, then use the string later. This one converts the path back to a string. Is that intentional?

Reply:
You're right.  This one was updated before I had updated the others and ran into a problem.

Given this use of ekopn(), we might want to re-consider our decision that "internal names" are strings.  Not sure how ekopn (or internal names) are used in general.